### DISCUSSION TITLE
AutoGluon Tabular with Multimodal

### DISCUSSION THREAD

**Discussion by Recep1997** [autogluon/autogluon#1781] [Q&A]
Hello dear users,

I wanted to replicate the tutorial from the official site and I get this error. Unfortunately I do not know how to work around it. Have actually recreated the codes from the tutorial exactly the same way.
<img width="795" alt="Bildschirmfoto 2022-06-05 um 14 25 24" src="https://user-images.githubusercontent.com/106914151/172050323-6e4058a0-42cd-4e8a-8dc7-37bb666961ff.png">

**Comment by Recep1997** [comment]
Link for the Tutorial: https://auto.gluon.ai/dev/tutorials/tabular_prediction/tabular-multimodal-text-others.html

**Reply by sxjscience** [reply]
The reason is that TextPredictor in tabular multimodal requires at least 1 GPU.

**Reply by Recep1997** [reply]
Thank you for your answer, the classic TextPredicter works on my Notebook so I thought that should work too. So you are saying that a computer with a GPU it should run normally? Sorry for my dumb questions, i am new in this section :(

**Reply by Recep1997** [reply]
I am using a MacBook Pro (Year 2016)

**Reply by sxjscience** [reply]
The root cause is these lines: https://github.com/awslabs/autogluon/blob/0ff9f8e555c371e53c4c613c2e1c5f023d953650/tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py#L205-L209 . You may change "num_gpus" from 1 to 0.

**Reply by Recep1997** [reply]
Unfortunately, I do not know. Where can I find this code section. So far I have only used the Spyder Notebook and imported the methods there. The methods I have all left on the standard :(

**Reply by sxjscience** [reply]
@Innixma Should we change the minimum_resources of TextPredictor in Tabular? Actually, TextPredictor also works for CPU-only instances.

**Reply by Innixma** [reply]
The logic errors in Tabular if no GPU because we previously had users that were training TextPredictor forever on CPU. Please use a GPU machine for training multimodal.
